### PR TITLE
Dummy pages created

### LINKED ISSUE #61: Frontend: Create Dummy Pages
Create Dummy Pages for the following Pages:

LoginPage - src/client/containers/LoginPage/LoginPage.component.js
SelectionPage - src/client/containers/SelectionPage/SelectionPage.component.js
RegistrationPage - src/client/containers/RegistrationPage/RegistrationPage.component.js
EventsPage - src/client/containers/EventsPage/EventsPage.component.js
ProfilePage - src/client/containers/ProfilePage/ProfilePage.component.js
404Page - src/client/containers/404Page/404Page.component.js

At this point will be used as placeholders so everybody will place the components already developed their.

Please only write a "Title" so everybody can understand what should be placed there.

### PR DESCRIPTION
**Pull request by Deepti-Dk**
# Description

Creates dummy pages (for naming consistency).

Fixes #61 

# How to test?

These are just empty pages with a respective title.

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by tpetrina — DISMISSED** [review]
Feel free to also add routes before merging.

**Review by nuha90 — DISMISSED** [review]

**Comment by Deepti-Dk** [comment]
> Feel free to also add routes before merging.

Done

**Review by nuha90 — APPROVED** [review]

File: src/client/App.js [hunk 1]
```diff
@@ -21,6 +27,23 @@ function App() {
         <Route exact path="/">
           <Home />
         </Route>
+
+        <Route exact path="/login-page">
```
Comment (tpetrina) [inline src/client/App.js:31]: We don't usually put `-page` in URLs, `/login` is fine.

Comment (Deepti-Dk) [reply src/client/App.js]: > We don't usually put `-page` in URLs, `/login` is fine.

updated URL names

**Review by tpetrina — APPROVED** [review]

**Review by nuha90 — APPROVED** [review]

**Pull request merged**